tools/codegen.mk

```diff
@@ -24,3 +24,15 @@ CONTROLLER_GEN=$(GOBIN)/controller-gen
 else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
+
+protoc-gen-internals-v1:
+	protoc -I . ./dapr/proto/internals/v1/*.proto --go_out=plugins=grpc:.
+	cp -R ./github.com/dapr/dapr/pkg/proto/internals/v1/*.go ./pkg/proto/internals/v1
```

Comment:
Please use the exact output path instead of copying the files: https://github.com/dapr/dapr/tree/master/dapr#generate-go-clients

Also this is the old way to generate stub clients, we have to use new way using new command - https://github.com/dapr/dapr/tree/master/dapr#generate-go-clients

Please see more details here - https://github.com/dapr/dapr/issues/2481

Reply:
Maybe we can generate pb files as is, then generate clients with the newer way.

Reply:
OK. Doing it.

Reply:
It is not possible to avoid copying. The command in the instruction below assumes users follow the `./github.com/...` structure locally for their source code development - which is a strong assumption. I did improve the Makefile to include all targets now. I also generated using the same protoc and proto-gen-go versions mentioned in the instructions.

Reply:
JFYI - The path was based on our official dev guide - https://github.com/dapr/dapr/blob/master/docs/development/developing-dapr.md

After using go.mod, contributors have not needed to use `GOPATH`.